tests/mobile/test_search.py

```diff
@@ -44,34 +37,12 @@ def test_that_searching_returns_results(self, mozwebqa):
                 Assert.equal(search_term, search_page.results[i].name)
 
     @pytest.mark.nondestructive
-    @pytest.mark.skipif('True', 'Test skipped because of Bug 847625 - Port search suggestions to Fireplace')
```

Comment:
This bug is now marked as WONTFIX so we have removed this test. I verified this with @krupa.